owapi/heroes.py

```diff
@@ -0,0 +1,279 @@
+"""
+This contains the static content for the heroes api endpoint.
+"""
+HEROES = {
+        "heroes": {
+            "offense": [
+                "genji",
+                "mccree",
+                "pharah",
+                "reaper",
+                "soldier76",
+                "sombra",
+                "tracer"
+            ],
+            "defense": [
+                "bastion",
+                "hanzo",
+                "junkrat",
+                "mei",
+                "torbjorn",
+                "widowmaker"
+            ],
+            "tank": [
+                "dva",
+                "reinhardt",
+                "roadhog",
+                "winston",
+                "zarya",
+            ],
+            "support": [
+                "ana",
+                "lucio",
+                "mercy",
+                "symmetra",
+                "zenyatta"
+            ]
+        }
+    }
+
+HERO = {
```

Comment:
This would be much better if it was scraped and autogenerated rather than having it hardcoded. Updating it every time a hero change is rolled out or a new hero is added is a pain.